internal/oem/oem.go

```diff
@@ -76,6 +77,7 @@ func (c Config) DefaultUserConfig() types.Config {
 }
 
 var configs = registry.Create("oem configs")
+var yes = util.BoolToPtr(true)
```

Comment:
Maybe drop this line entirely and do `util.BoolToPtr(true)` everywhere below, but I'm rather indifferent about it.

LGTM.